arcgis-ios-sdk-samples/Maps/Download preplanned map/DownloadPreplannedMap.storyboard

```diff
@@ -0,0 +1,155 @@
+                                            <rect key="frame" x="20" y="0.0" width="374" height="44"/>
+                                            <autoresizingMask key="autoresizingMask"/>
+                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                    </subviews>
+                                </tableViewCellContentView>
+                            </tableViewCell>
+                            <tableViewCell clipsSubviews="YES" contentMode="scaleToFill" insetsLayoutMarginsFromSafeArea="NO" selectionStyle="default" hidesAccessoryWhenEditing="NO" indentationLevel="1" indentationWidth="0.0" reuseIdentifier="PreplannedMapAreaCell" textLabel="uxQ-9t-3gv" style="IBUITableViewCellStyleDefault" id="n2M-d9-GMN" customClass="PreplannedMapAreaTableViewCell" customModule="ArcGIS_Runtime_SDK_Samples" customModuleProvider="target">
+                                <rect key="frame" x="0.0" y="99.5" width="414" height="44"/>
+                                <autoresizingMask key="autoresizingMask"/>
+                                <tableViewCellContentView key="contentView" opaque="NO" clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="center" insetsLayoutMarginsFromSafeArea="NO" tableViewCell="n2M-d9-GMN" id="SZi-4t-izK">
+                                    <rect key="frame" x="0.0" y="0.0" width="414" height="44"/>
+                                    <autoresizingMask key="autoresizingMask"/>
+                                    <subviews>
+                                        <label opaque="NO" multipleTouchEnabled="YES" contentMode="left" insetsLayoutMarginsFromSafeArea="NO" text="Preplanned Area" lineBreakMode="tailTruncation" baselineAdjustment="alignBaselines" adjustsFontSizeToFit="NO" id="uxQ-9t-3gv">
+                                            <rect key="frame" x="20" y="0.0" width="374" height="44"/>
+                                            <autoresizingMask key="autoresizingMask"/>
+                                            <fontDescription key="fontDescription" type="system" pointSize="17"/>
+                                            <nil key="textColor"/>
+                                            <nil key="highlightedColor"/>
+                                        </label>
+                                        <progressView opaque="NO" contentMode="scaleToFill" verticalHuggingPriority="750" id="lSk-Pe-aUi">
+                                            <rect key="frame" x="0.0" y="42" width="414" height="2"/>
+                                            <autoresizingMask key="autoresizingMask" flexibleMinY="YES"/>
```

Comment:
I would think that we'd want the width to expand/contract to fit the content view's width:
```suggestion
                                            <autoresizingMask key="autoresizingMask" widthSizable="YES" flexibleMinY="YES"/>
```
The only downside I can think of is that the width of the progress view of the selected cell will be smaller since the accessory is outside of the content view:

<img width="376" alt="image" src="https://user-images.githubusercontent.com/2257493/63556497-f2262e00-c4f9-11e9-942c-aedab84dd609.png">

But I think I'd prefer that over using the width defined in the storyboard, which may or may not be the width of the content view on screen. What do you think?